mod-source-record-storage-server/src/main/java/org/folio/consumers/DataImportKafkaHandler.java

```diff
@@ -68,22 +70,21 @@ public DataImportKafkaHandler(Vertx vertx, JobProfileSnapshotCache profileSnapsh
           } else if (DI_ERROR.value().equals(processedPayload.getEventType())) {
             promise.fail("Failed to process data import event payload");
           } else {
-            promise.complete(record.key());
+            promise.complete(targetRecord.key());
           }
         });
       return promise.future();
     } catch (Exception e) {
-      LOGGER.error("Failed to process data import kafka record from topic {}", record.topic(), e);
+      LOGGER.error("Failed to process data import kafka record from topic {}", targetRecord.topic(), e);
```

Comment:
It might make sense to add chunkId and recordId here?

Reply:
Done